### DISCUSSION TITLE
Help with replacing character on removal instead of shifting left

### DISCUSSION THREAD

**Discussion by havardthom** [uNmAnNeR/imaskjs#716] [Q&A] [answered]
Hello, hoping to get some help here. 
I'm looking for a way to replace removed characters with a default value, instead of the remaining characters shifting left. 

e.g. 
If i have `12° 34' 56" N`
and i remove the number 3
i want to get `12° 04' 56" N` (number 3 is replaced by a default value 0)
instead of `12° 45' _6" N` (current behaviour where characters shifts left on removal)

You could say i want all characters in the input to be required, so removing a character should not be possible, it should be filled in by a default value.

Code example:
https://codesandbox.io/s/vanilla-forked-g8c36i

**Comment by uNmAnNeR** [comment, marked as answer]
using `backquotes` can help not to shift between blocks like
```
'`dd° `mm\' `ss" `dir'
```

But for inside blocks for now i have no idea how to prevent shifting.
Thoughts on it:
`mm` pattern becomes `00` mask with validators.
Fix could be using `'0'0`. So probably need some way to add additional options. like:
```
mm: {
    mask: IMask.MaskedRange,
    from: 0,
    to: 59,
    numberChar: '`0',
  },
```

but it looks ugly.

For you case i suggest not to use Masked range, but just masked pattern ('`0`0') with `validate` option
